requirements.txt

```diff
@@ -17,16 +17,19 @@ fastapi-sqlalchemy==0.2.1 ; python_version >= "3.11" and python_version < "4.0"
 fastapi==0.103.2 ; python_version >= "3.11" and python_version < "4.0"
 greenlet==3.0.2 ; python_version >= "3.11" and python_version < "4.0" and (platform_machine == "aarch64" or platform_machine == "ppc64le" or platform_machine == "x86_64" or platform_machine == "amd64" or platform_machine == "AMD64" or platform_machine == "win32" or platform_machine == "WIN32")
 h11==0.14.0 ; python_version >= "3.11" and python_version < "4.0"
+httpcore==1.0.2 ; python_version >= "3.11" and python_version < "4.0"
 httptools==0.6.1 ; python_version >= "3.11" and python_version < "4.0"
+httpx==0.26.0 ; python_version >= "3.11" and python_version < "4.0"
 idna==3.6 ; python_version >= "3.11" and python_version < "4.0"
 iniconfig==2.0.0 ; python_version >= "3.11" and python_version < "4.0"
 jinja2==3.1.2 ; python_version >= "3.11" and python_version < "4.0"
 mako==1.3.0 ; python_version >= "3.11" and python_version < "4.0"
 markupsafe==2.1.3 ; python_version >= "3.11" and python_version < "4.0"
+numpy==1.26.2 ; python_version >= "3.12" and python_version < "4.0" or python_version == "3.11"
 outcome==1.3.0.post0 ; python_version >= "3.11" and python_version < "4.0"
 packaging==23.2 ; python_version >= "3.11" and python_version < "4.0"
+pandas==2.1.4 ; python_version >= "3.11" and python_version < "4.0"
```

Comment:
To add a new package, @anisfeld, you should do the `poetry add __` statement that's in the README and then run `make create_requirements` and you'll have your updated requirements.txt file: https://github.com/prijatelilab/PrijateliTree/pull/119

Reply:
Thanks! I was rushing b/c we were meeting with our partners this morning. I appreciate the pointer!

Reply:
I figured as much! Hopefully this is helpful.